stable/memcached/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+name: memcached
+version: 0.4.0
+description: Chart for Memcached
+keywords:
+- memcached
+- cache
+home: http://memcached.org/
+sources:
+- https://github.com/docker-library/memcached
+maintainers:
+- name: Bitnami
+  email: containers@bitnami.com
```

Comment:
@gtaylor First off, nice work on this chart :slightly_smiling_face: 

Bitnami tests and validates its helm charts against the [Bitnami docker images](https://hub.docker.com/u/bitnami/). Since this chart does not use the `bitnami/memcached` image we would not be offering support/maintaining this chart as Bitnami. I'd suggest removing Bitnami from the maintainers list and instead attribute the [Bitnami Charts repo](https://github.com/bitnami/charts) in the readme as @prydonius mentioned.

Out of curiosity, was there any particular reason for you to swap out the `bitnami/memcached` image? At Bitnami we are heavily invested in offering users the best and up-to-date docker images and helm charts. Your feedback would help us improve our offerings.